reqhandler/fields.go

```diff
@@ -0,0 +1,33 @@
+package reqhandler
+
+type FieldName string
+
+const (
+	ID       FieldName = "id"
+	QUERY              = "query"
```

Comment:
Why do you need each header part to map to a string? Why not use [iota](https://github.com/golang/go/wiki/Iota)?